inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp

```diff
@@ -132,16 +132,27 @@ void FrontEnd::parseRNN(const Model& model, const ie::CNNLayerPtr& _layer, const
 
     size_t nCells = inputs[0]->desc().dim(Dim::H);
     size_t nBatches = inputs[0]->desc().dim(Dim::C);
+    size_t inputSize = inputs[0]->desc().dim(Dim::W);
 
-    if (outputs.size() == 3) {
-        nCells = inputs[0]->desc().dim(Dim::C);
+    int axis = 1; // TODO : get from xml
```

Comment:
take a look at GPU plugin implementation:
inference-engine/src/cldnn_engine/cldnn_lstm.cpp

they process this parameter as:
`    bool permute_input = (1 != rnnLayer->axis);
`
...

        if (!permute_input) {
            lstm_batch_size = in_dims0.front();
            lstm_sequence_len = in_dims0[1];
        } else {
            lstm_batch_size = in_dims0[1];
            lstm_sequence_len = in_dims0.front();
        }
`
you can do similar thing.